<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preferences</class>
 <widget class="QDialog" name="Preferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resource/ribbon_resource.qrc">
    <normaloff>:/image/image/sc_v12.png</normaloff>:/image/image/sc_v12.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="lineEdit_search">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
selection-background-color: rgb(0, 0, 0);
selection-color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-color:rgb(159, 159, 159);
border: 1px solid rgb(0, 0, 0);</string>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="treeWidget">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-color:rgb(159, 159, 159);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="headerHidden">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>First</string>
          </property>
         </column>
         <item>
          <property name="text">
           <string>Appearance</string>
          </property>
          <item>
           <property name="text">
            <string>Appearance</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>System Settings</string>
           </property>
          </item>
         </item>
         <item>
          <property name="text">
           <string>Keymap</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Editor</string>
          </property>
          <item>
           <property name="text">
            <string>General</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Font</string>
           </property>
          </item>
         </item>
         <item>
          <property name="text">
           <string>Set Paths</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Plugins</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-color: rgb(159, 159, 159);
</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page_appearance"/>
       <widget class="QWidget" name="page_system_setting"/>
       <widget class="QWidget" name="page_keymap"/>
       <widget class="QWidget" name="page_general">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QGroupBox" name="groupBox_editor_general">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">#groupBox_editor_general{border:none}</string>
           </property>
           <property name="title">
            <string>Editor &gt; General</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5" stretch="3">
            <property name="topMargin">
             <number>30</number>
            </property>
            <item>
             <widget class="QGroupBox" name="groupBox_3">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="title">
               <string>Others</string>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_font"/>
       <widget class="QWidget" name="page_plugins"/>
       <widget class="QWidget" name="page_set_paths">
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QGroupBox" name="groupBox_setPaths">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">#groupBox_setPaths{border:none}</string>
           </property>
           <property name="title">
            <string>Set Paths</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <property name="topMargin">
             <number>30</number>
            </property>
            <item>
             <widget class="QGroupBox" name="groupBox_editor_paths">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="title">
               <string>Editor Paths</string>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <layout class="QGridLayout" name="gridLayout">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>VS Code：</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLineEdit" name="lineEdit_vscode">
                   <property name="styleSheet">
                    <string notr="true">border: 1px solid rgb(0, 0, 0);</string>
                   </property>
                   <property name="frame">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="2">
                  <widget class="QToolButton" name="btn_vscode">
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset resource="../../resource/ribbon_resource.qrc">
                     <normaloff>:/image/image/arrow-down-circle.png</normaloff>:/image/image/arrow-down-circle.png</iconset>
                   </property>
                   <property name="autoRaise">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>Notepad++：</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLineEdit" name="lineEdit_notepad">
                   <property name="styleSheet">
                    <string notr="true">border: 1px solid rgb(0, 0, 0);</string>
                   </property>
                   <property name="frame">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="2">
                  <widget class="QToolButton" name="btn_notepad">
                   <property name="text">
                    <string>Notepad++</string>
                   </property>
                   <property name="icon">
                    <iconset resource="../../resource/ribbon_resource.qrc">
                     <normaloff>:/image/image/arrow-down-circle.png</normaloff>:/image/image/arrow-down-circle.png</iconset>
                   </property>
                   <property name="autoRaise">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>Sublime Text：</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QLineEdit" name="lineEdit_subl">
                   <property name="styleSheet">
                    <string notr="true">border: 1px solid rgb(0, 0, 0);</string>
                   </property>
                   <property name="frame">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="2">
                  <widget class="QToolButton" name="btn_subl">
                   <property name="text">
                    <string>Notepad++</string>
                   </property>
                   <property name="icon">
                    <iconset resource="../../resource/ribbon_resource.qrc">
                     <normaloff>:/image/image/arrow-down-circle.png</normaloff>:/image/image/arrow-down-circle.png</iconset>
                   </property>
                   <property name="autoRaise">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,10,0,0,0">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QToolButton" name="btn_help">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../resource/ribbon_resource.qrc">
         <normaloff>:/image/image/help.png</normaloff>:/image/image/help.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>16</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_ok">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>ok</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_apply">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resource/ribbon_resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_ok</sender>
   <signal>clicked()</signal>
   <receiver>Preferences</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>513</x>
     <y>466</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_cancel</sender>
   <signal>clicked()</signal>
   <receiver>Preferences</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>594</x>
     <y>466</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_apply</sender>
   <signal>clicked()</signal>
   <receiver>Preferences</receiver>
   <slot>exec()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>675</x>
     <y>466</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
